troposphere/static/js/AppRoutes.react.js

```diff
@@ -76,7 +77,8 @@ define(function (require) {
       <Route name="settings" handler={SettingsPage}/>
 
       <Route name="admin" handler={AdminMaster}>
-        <DefaultRoute name="identity-membership-manager" handler={IdentityMembershipMaster}/>
+        <DefaultRoute name="atmosphere-user-manager" path="users" handler={AtmosphereUserMaster}/>
```

Comment:
So, what if i *want* the users path to be in the URL, and also for it to be the DefaultRoute, would the proper syntax then be ...
`<Route ...>`
...
`<DefaultRoute>`?